v0.12.1 crashes upon startup

Specifically with `UnsatisfiedLinkError: ... couldn't find "libgdx.so"`

This is due to a bump in the gradle version and gradle plugin version. I attempted to fix it in v0.12.1 but obviously have not succeeded. For reference:
* [Discussion on the Unciv issue tracker](https://github.com/yairm210/Unciv/issues/3946#issuecomment-857474194).
* [Fix from the Unciv team](https://github.com/yairm210/Unciv/pull/4090/files)
* [Same issue on retrowars](https://github.com/retrowars/retrowars/issues/25)
* [Fix for retrowars](https://github.com/retrowars/retrowars/commit/27dc3573d0cecf3fd57f3b63272e41351a3f1b74)

This is the logcat error from Beat Feet:

```
07-14 06:57:34.954 22700 22700 E AndroidRuntime: FATAL EXCEPTION: main
07-14 06:57:34.954 22700 22700 E AndroidRuntime: Process: com.serwylo.beatgame, PID: 22700
07-14 06:57:34.954 22700 22700 E AndroidRuntime: java.lang.ExceptionInInitializerError
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at java.lang.Class.newInstance(Native Method)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:95)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.Instrumentation.newActivity(Instrumentation.java:1264)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3797)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4077)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:91)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2458)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:110)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:219)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8387)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1055)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load shared library 'gdx' for target: Linux, 32-bit
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at com.badlogic.gdx.utils.SharedLibraryLoader.load(SharedLibraryLoader.java:126)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at com.badlogic.gdx.utils.GdxNativesLoader.load(GdxNativesLoader.java:31)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at com.badlogic.gdx.backends.android.AndroidApplication.<clinit>(AndroidApplication.java:44)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	... 15 more
07-14 06:57:34.954 22700 22700 E AndroidRuntime: Caused by: java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.serwylo.beatgame-0mWSTSF-c2Ly2L-RJdNrfg==/base.apk"],nativeLibraryDirectories=[/data/app/com.serwylo.beatgame-0mWSTSF-c2Ly2L-RJdNrfg==/lib/arm64, /system/lib64, /hw_product/lib64, /system/product/lib64, /prets/lib64]]] couldn't find "libgdx.so"
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at java.lang.Runtime.loadLibrary0(Runtime.java:1067)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at java.lang.Runtime.loadLibrary0(Runtime.java:1007)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at java.lang.System.loadLibrary(System.java:1668)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	at com.badlogic.gdx.utils.SharedLibraryLoader.load(SharedLibraryLoader.java:120)
07-14 06:57:34.954 22700 22700 E AndroidRuntime: 	... 17 more
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -20,8 +20,8 @@ android {
         applicationId "com.serwylo.beatgame"
         minSdkVersion 14
         targetSdkVersion 30
-        versionCode 14
-        versionName "0.12.1"
+        versionCode 15
+        versionName "0.12.2"
     }
     buildTypes {
         release {
@@ -66,7 +66,7 @@ tasks.whenTaskAdded { packageTask ->
     // See:
     //  * https://github.com/yairm210/Unciv/issues/3946 for the problem, and
     //  * https://github.com/yairm210/Unciv/pull/4090/files for the solution
-    if (packageTask.name.contains("package")) {
+    if (packageTask.name.contains("package") || packageTask.name.contains("assemble")) {
         packageTask.dependsOn 'copyAndroidNatives'
     }
 }

--- a/fastlane/metadata/android/de/changelogs/15.txt
+++ b/fastlane/metadata/android/de/changelogs/15.txt
@@ -1,0 +1,18 @@
+👍 Mehr Übersetzungen!
+
+* Neu (Dank an):
+ - Esperanto (chtabs2k19)
+ - Indonesisch (liimee + rezaalmanda)
+ - Portugiesisch (larcansal)
+ - Russisch (chtabs2k19)
+ - Vietnamesisch (quangtrung02hn16)
+
+* Neue Schriftart aktiviert:
+  - Mazedonisch (primarto24c8a9c6889c407b)
+  - Polnisch (Michaldariusznowakowski)
+
+* Leider werden diese noch nicht unterstützt (fehlende Richtung- oder Font-Unterstützung):
+  - Bengalisch (danke Oymate)
+  - Persisch (danke Majid)
+
+Unterstütze uns über GitHub oder Liberapay.

--- a/fastlane/metadata/android/en-US/changelogs/15.txt
+++ b/fastlane/metadata/android/en-US/changelogs/15.txt
@@ -1,0 +1,18 @@
+👍 More translations!
+
+* New (thanks to):
+ - Esperanto (chtabs2k19)
+ - Indonesian (liimee + rezaalmanda)
+ - Portuguese (larcansal)
+ - Russian (chtabs2k19)
+ - Vietnamese (quangtrung02hn16)
+
+* A new font has enabled:
+  - Macedonian (primarto24c8a9c6889c407b)
+  - Polish (Michaldariusznowakowski)
+
+* Unfortunately, these are still unsupported (lack of RTL or font support):
+  - Bengali (thanks Oymate)
+  - Persian (thanks majid)
+
+Support further development via GitHub Sponsors or Liberapay.

--- a/fastlane/metadata/android/fr/changelogs/15.txt
+++ b/fastlane/metadata/android/fr/changelogs/15.txt
@@ -1,0 +1,18 @@
+👍 Plus de traductions !
+
+* Nouveau (grâce à) :
+ - Espéranto (chtabs2k19)
+ - Indonésien (liimee + rezaalmanda)
+ - Portugais (larcansal)
+ - Russe (chtabs2k19)
+ - Vietnamien (quangtrung02hn16)
+
+* Nouvelle police :
+  - Macédonien (primarto24c8a9c6889c407b)
+  - Polonais (Michaldariusznowakowski)
+
+* Malheureusement, ces langues ne sont toujours pas prises en charge (problème de direction ou de police) :
+  - Bengali (merci Oymate)
+  - Persan (merci majid)
+
+Soutenez le projet via GitHub ou Liberapay.

--- a/fastlane/metadata/android/nb-NO/changelogs/15.txt
+++ b/fastlane/metadata/android/nb-NO/changelogs/15.txt
@@ -1,0 +1,18 @@
+👍 Flere oversettelser!
+
+* Nye (takket være):
+ - Esperanto (chtabs2k19)
+ - Indonesisk (liimee + rezaalmanda)
+ - Portugisisk (larcansal)
+ - Russisk (chtabs2k19)
+ - Vietnamesisk (quangtrung02hn16)
+
+* Ny skrift:
+  - Makedonsk (primarto24c8a9c6889c407b)
+  - Polsk (Michaldariusznowakowski)
+
+* Beklageligvis støttes ikke disse (på grunn av mangel på VTH-støtte):
+  - Bengalsk (Oymate)
+  - Persisk (majid)
+
+Støtt videre utvikling på Liberapay eller GitHub Sponsors.

--- a/fastlane/metadata/android/pt/changelogs/15.txt
+++ b/fastlane/metadata/android/pt/changelogs/15.txt
@@ -1,0 +1,18 @@
+👍 Mais traduções!
+
+* Novo (graças a):
+ - Esperanto (chtabs2k19)
+ - Indonésio (liimee + rezaalmanda)
+ - Português (larcansal)
+ - Russo (chtabs2k19)
+ - Vietnamita (quangtrung02hn16)
+
+* Ativada uma nova fonte:
+  - Macedónio (primarto24c8a9c6889c407b)
+  - Polaco (Michaldariusznowakowski)
+
+* Infelizmente, estes ainda não são suportados (falta de RTL ou suporte de fontes):
+  - Bengali (obrigado Oymate)
+  - Persa (obrigado majid)
+
+Apoie o desenvolvimento no GitHub Sponsors ou Liberapay.

